front/app/api/comments/useMarkForDeletion.ts

```diff
@@ -13,9 +13,6 @@ interface MarkForDeletion {
 }
 
 const markForDeletion = async ({ commentId, reason }: MarkForDeletion) => {
-  if (reason && reason.reason_code !== 'other') {
-    reason.other_reason = null;
-  }
```

Comment:
No need to set it to null. Changed `other_reason` type to potentially include undefined.